apps/charterafrica/src/payload/collections/Media.js

```diff
@@ -19,6 +19,14 @@ const Media = {
       required: true,
     },
   ],
+  hooks: {
+    afterRead: [
+      ({ doc }) => ({
+        ...doc,
+        alt: doc.alt || doc.filename,
+      }),
+    ],
+  },
```

Comment:
Makes sense-ish but not too sure about it... isn't it something that should be decided on front-end? e.g. no alt, use org's or tool's name instead, etc.

Reply:
@kilemensi this approach ensures we do not get `undefined` serializing errors on development.

Reply:
Yes @koechkevin but that means the front-end won't know if the user actually wanted to use the `alt` or not. `undefined` can't be serialized but `null` can.